Assets/Scripts/UI/Overlay/OverlayMap.cs

```diff
@@ -324,6 +324,8 @@ private void Start()
         
         script = new Script();
         script.DoString(scriptTxt);
+        Type type = typeof(ModUtils);
+        script.Globals[type.Name] = type;
```

Comment:
This whole file should probably be changed to use `LuaUtilities`, but that's for another PR. Maybe stick a TODO in here somewhere?